src/test/test_Scanner.cxx

```diff
@@ -59,7 +59,8 @@ run_tests()
   Scanner::Type type= Scanner::E931; 
   while (type != Scanner::Unknown_scanner)
   {
-    if (type!=Scanner::User_defined_scanner)
+    if (type!=Scanner::User_defined_scanner &&
+            type!=Scanner::UPENN_5rings)
```

Comment:
@KrisThielemans I know that you might not like this, but I don't have a better solution

Reply:
Yes, but I need the virtual crystals after each scanner's block (head). Not the smaller "read out" blocks. If that makes sense.

Reply:
A single ring has 576 detectors and I want to add 18 gaps, one per block, the new total is 594 detectors per ring.

Reply:
is the problem that you're adding a gap at the end? Not sure why you'd do that, but potentially could adapt test_scanner to cope with that? might not be worth it, but I'd like to understand

Reply:
The consistency check tries to get the number of blocks from the total number of detectors per ring, but when that is 594 then it does not find 18 blocks and throughs an error.

Reply:
Currently, `num_detectors_per_ring()`, `num_crystals_per_block` etc include the virtual crystals. So I believe that the transaxial check should work. Did you set `get_num_virtual_axial_crystals_per_block` and `get_num_virtual_transaxial_crystals_per_block`?

See also #629

Reply:
The latest commit should address this issue

Reply:
I reverted this part of the code and made the Scanner definitions such that the test_Scanner passes (at least in my laptop). However, I have to note that the definitions, do not really make sense now.

Reply:
oh well. let's leave this for later